lib/util/addExtraParamsTo.js

```diff
@@ -10,7 +10,11 @@ function addExtraParamsTo(routeArguments) {
     //  is null. WWARNER 08/21/2014
     paramsAndRequest.pop();
 
-    var extraParams = Array.prototype.slice.call(arguments, 1);
```

Comment:
yeah because arguments from this function is passed into the other function body. even though slice will make a shallow copy, we need to avoid passing this function's arguments to it